test/test_jit_cuda_fuser.py

```diff
@@ -588,7 +595,8 @@ def test_unary_ops(self):
             self._unary_test_helper(op, dtype, False)  # test special numbers
             self._unary_test_helper(op, dtype, True)  # test random data
 
-    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    @unittest.skipIf(True, "See issues 73395 and 75029")
```

Comment:
hmmmm.. I don't remember seeing test failures in my last upstream push. let me take a look here.